modules/tinymce/src/core/main/ts/fmt/DefaultFormats.ts

```diff
@@ -167,11 +170,12 @@ const get = (dom: DOMUtils) => {
       { inline: 'u', remove: 'all', preserve_attributes: [ 'class', 'style' ] }
     ],
 
-    strikethrough: [
-      { inline: 'span', styles: { textDecoration: 'line-through' }, exact: true },
-      { inline: 'strike', remove: 'all', preserve_attributes: [ 'class', 'style' ] },
-      { inline: 's', remove: 'all', preserve_attributes: [ 'class', 'style' ] }
-    ],
+    strikethrough: (() => {
```

Comment:
Is this a style we prefer having a anonymous function immediately invoked like this. At lest it's not something I've seen used a lot. It could instead be just a function above the get call.

```
const strikethrough = (schemaType: string): Format => {
  const span: Format = { inline: 'span', styles: { textDecoration: 'line-through' }, exact: true };
  const strike: Format = { inline: 'strike', remove: 'all', preserve_attributes: [ 'class', 'style' ] };
  const s: Format = { inline: 's', remove: 'all', preserve_attributes: [ 'class', 'style' ] };
  return schemaType !== 'html4' ? [ s, span, strike ] : [ span, s, strike ];
});
```

Then at the call site be `strikethough: strikethough(editor.schema.type)`. To me at least that is easier to read. But I guess it's a matter of style.

Reply:
I did it this way so that you could still see all of the format configs together without having to look at a separate function in a different location. I don't have any strong opinions so if others think the other way is preferable I am happy to go with that.

Reply:
It's unusual, but it's the only way to collect a bunch of variables without adding them to the scope. Or, as you said, make it a function nearby.

I don't mind too much which way we decide to go.

Reply:
We discussed it post-standup and are happy to go with the current inline function setup